test/alternator/conftest.py

```diff
@@ -69,6 +93,10 @@ def dynamodb(request):
     # Disable boto3's client-side validation of parameters. This validation
     # only makes it impossible for us to test various error conditions,
     # because boto3 checks them before we can get the server to check them.
+    host = request.config.getoption('host')
+    cluster = get_cql_cluster(host)
+    cluster.connect().execute(
+        "INSERT INTO system_auth_v2.roles (role, salted_hash) VALUES ('alternator', 'secret_pass')")
```

Comment:
this logic should be locatae where the cluster is created (run test.py)

if you want to eliminate run you will have to move all logic of cluster creation to pytest. that I think, is too much for this task